scripts/generator/graphql_generator/csharp/Editor/ShopifyIosPostProcessor.cs.erb

```diff
@@ -0,0 +1,43 @@
+namespace <%= namespace %>.SDK {
+    using UnityEngine;
+    using UnityEditor;
+    using UnityEditor.iOS.Xcode;
+    using System.IO;
+    using System;
+
+    public class ShopifyIosPostProcessor {
+        public static void Process(string buildPath) {
+            string testPath = Path.Combine(buildPath, "Libraries/Plugins/iOS/Shopify/BuyTests/");
+            var testDirectory = new DirectoryInfo(testPath);
+            var project = new ExtendedPBXProject(buildPath);
+
+            try {
+                project.SetFilesInDirectoryToTestTarget(testDirectory);
+            } catch (Exception e) {
+                Debug.Log(e.Message);
+            }
+
+            // Removes default Unity_iPhone_Tests from the Test Target
+            project.RemoveFileFromBuild(project.TestTargetGuid, project.FindFileGuidByProjectPath("Unity-iPhone Tests/Unity_iPhone_Tests.m"));
```

Comment:
I suppose for now we can remove having this, till we implement automated test cases for our plugin. Perhaps we can have this added specifically for the CI. It would only affect those that have modified the file directly.